src/orchestrator-tracing/Cargo.toml

```diff
@@ -12,9 +12,10 @@ async-trait = "0.1.56"
 clap = { version = "3.1.18", features = ["env", "derive"] }
 http = "0.2.7"
 mz-orchestrator = { path = "../orchestrator" }
-mz-ore = { path = "../ore" }
+mz-ore = { path = "../ore", features = ["tracing_"] }
```

Comment:
because `tracing` is a crate name so it conflicts, we could use `dep:tracing` inside `mz-ore` to fix this, but dependenbot doesnt work with `dep:` yet!